hub/core/chunk/uncompressed_chunk.py

```diff
@@ -32,7 +32,7 @@ def _extend_if_has_space_numpy(self, incoming_samples: np.ndarray) -> float:
 
         samples = incoming_samples[:num_samples]
         samples = intelligent_cast(samples, self.dtype, self.htype)
-        self.data_bytes += samples.tobytes()
+        self.data_bytes += samples.tobytes()  # type: ignore
```

Comment:
why are ignoring type here?

Reply:
data_bytes is a bytearray by this point due to earlier code but mypy still considers it a union of PartialReader and bytes which is why type needs to be ignored

Reply:
can you make tobytes also be a union of bytearray?

Reply:
I think we should try to make it as strongly typed as possible.